dsls/sql/src/main/java/org/apache/beam/dsls/sql/example/BeamSqlExample.java

```diff
@@ -58,16 +60,31 @@ public static void main(String[] args) throws Exception {
     PCollection<BeamSQLRow> inputTable = PBegin.in(p).apply(Create.of(row)
         .withCoder(new BeamSqlRowCoder(type)));
 
-    //run a simple SQL query over input PCollection;
-    String sql = "select c2, c3 from TABLE_A where c1=1";
-    PCollection<BeamSQLRow> outputStream = inputTable.apply(BeamSql.simpleQuery(sql));
+    //Case 1. run a simple SQL query over input PCollection with BeamSql.simpleQuery;
+    PCollection<BeamSQLRow> outputStream = inputTable.apply(
+        BeamSql.simpleQuery("select c2, c3 from TABLE_A where c1=1"));
```

Comment:
@takidau as talking about the design of interface, the default method is `public static PTransform<PCollectionTuple, PCollection<BeamSqlRow>> query(String sqlQuery)`, which relies on the named `TupleTag` to specify table name. 

`public static PTransform<PCollection<BeamSqlRow>, PCollection<BeamSqlRow>>
  simpleQuery(String sqlQuery)` is a special case which runs on single table/PCollection. There's no exiting method to name it, so the table name in query is take as granted.

For both methods, potentially the table name would be mixed up, that's why I said a further task is needed to have a separated schema namespace for each query. 

@lukecwik  any comments?